Prac_5/emails.py

```diff
@@ -0,0 +1,20 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:   30 minutes
+"""
+full_info_list = {}
```

Comment:
using the word list in the variable name might makes the reader think that it might be a list?